Apps/Sandcastle/gallery/development/Polylines.html

```diff
@@ -99,6 +99,24 @@
         material : Cesium.Material.fromType(Cesium.Material.PolylineArrowType)
     });
     Sandcastle.declare(localPolyline); // For highlighting on mouseover in Sandcastle.
+    
+    //Polyline using the fade material
+    var fadingPolyline = polylines.add({
+        positions : Cesium.Cartesian3.fromDegreesArrayHeights([-75, 43, 500000,
+                                                               -125, 43, 500000]),
+        width : 5,
+        material : Cesium.Material.fromType(Cesium.Material.FadeType, {
+            repeat: false,
+            fadeInColor: Cesium.Color.BLUE,
+            fadeOutColor: Cesium.Color.TRANSPARENT,
```

Comment:
So `TRANSPARENT` is actually implicitly a transparent `BLACK` color.  Instead, use `Cesium.Color.CYAN.withAlpha(0)`.

@hpinkos I know from our logo work that you have a good eye for shades of color.  Check out the difference with and without this change.